src/Stryker.Core/Stryker.Core/Reporters/StatusReporter.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using Crayon;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Stryker.Core.Mutants;
+using Stryker.Core.Options;
+using Stryker.Core.ProjectComponents;
+using Microsoft.Extensions.Logging;
+using Stryker.Core.Logging;
+
+namespace Stryker.Core.Reporters
+{
+    public class StatusReporter : IReporter
+    {
+        private readonly StrykerOptions _options;
+        private readonly ILogger<StatusReporter> _logger;
+        private readonly TextWriter _consoleWriter;
+
+
+        public StatusReporter(StrykerOptions strykerOptions, ILogger<StatusReporter> logger = null, TextWriter consoleWriter = null)
+        {
+            _options = strykerOptions;
+            _logger = logger ?? ApplicationLogging.LoggerFactory.CreateLogger<StatusReporter>();
+            _consoleWriter = consoleWriter ?? Console.Out;
```

Comment:
You are not using the console writer right? So could you remove it from the constructor?

Reply:
Oh right, forgot to remove it, thanks!